tools/IceRpc.Slice.Tools/IceRpc.Slice.Tools.props

```diff
@@ -20,12 +20,12 @@
         <SliceCompilerPath
           Condition="'$(Configuration)' == 'Release'"
         >$(MSBuildThisFileDirectory)../slicec-cs/target/release</SliceCompilerPath>
-        <IceRpcBuildTelemetryScriptPath
-        >$(MSBuildThisFileDirectory)../IceRpc.BuildTelemetry.Reporter/bin/$(Configuration)/net8.0/</IceRpcBuildTelemetryScriptPath>
+        <!-- Build telemetry is not supported with source builds -->
```

Comment:
I would update the comment to:
 Disable build telemetry when using a source build of slicec-cs